packages/plugins/slugManager/manager.ts

```diff
@@ -6,41 +6,41 @@ interface ProfileUpdate {
   [key: string]: string | boolean | Meta;
 }
 
+export type SlugFormatType =
+  | string
+  | {
+      generatorFn: (lang: string, profile: BaseEntity) => string;
+      fields?: string[];
```

Comment:
I think this should be required if someone's using a generatorFn, even if that means they have to explicitly provide an empty array if they're not using any fields.